ern-container-gen-android/src/hull/lib/src/main/java/com/walmartlabs/ern/container/ElectrodeReactContainer.java

```diff
@@ -136,6 +137,10 @@ public synchronized static void initialize(@NonNull Application application, @No
             {{/apiImplementations}}
      ) {
         if (sElectrodeReactNativeHost == null) {
+            
+            {{#RN_VERSION_GTE_60_1}}
+            SoLoader.init(application, /* native exopackage */ false);
+            {{/RN_VERSION_GTE_60_1}}
```

Comment:
Just double checking: This is _not_ required for React Native 0.60.0?

Reply:
Yes correct. Change was only introduced starting with RN 0.60.1 (for Hermes support)

Reply:
Yes, you're right, it won't crash below RN 0.60.1 - I'm just wondering if there could be any other unintentional effects by _not_ calling it? It appears the call to `SoLoader.init` has been there for a long time in the RN templates (at least all the way back to React Native `0.53.0`). Are there any disadvantages to simply always call this method?